sdk/frontdoor/Azure.ResourceManager.FrontDoor/api/Azure.ResourceManager.FrontDoor.netstandard2.0.cs

```diff
@@ -549,10 +432,10 @@ public partial class CustomRule
         public static bool operator !=(Azure.ResourceManager.FrontDoor.Models.DynamicCompressionEnabled left, Azure.ResourceManager.FrontDoor.Models.DynamicCompressionEnabled right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class Endpoint
+    public partial class EndpointProperties
     {
-        public Endpoint() { }
-        public string EndpointValue { get { throw null; } set { } }
+        public EndpointProperties() { }
+        public string Endpoint { get { throw null; } set { } }
```

Comment:
Should this be a URI?

Reply:
The sample in the swagger is `endpointA.net`, which looks like a relative uri, therefore I didn't change the type to avoid failure in deserialization.